crud.py

```diff
@@ -0,0 +1,58 @@
+from datetime import date
+
+from sqlalchemy.orm import Session
+
+import schemas
+from db.models import Author, Book
+
+
+def get_all_authors(db: Session):
+    return db.query(Author).all()
+
+
+def get_author_by_name(db: Session, name: str):
+    return db.query(Author).filter(Author.name == name).all()
```

Comment:
1. don't see where you use this function
2. Also don't see this requirement implemented "Retrieve a single author by ID."